src/main/java/bacnet/DeviceFolder.java

```diff
@@ -599,4 +576,29 @@ public DeviceFolder getMe() {
 		return this;
 	}
 
+	public RemoteDevice getRemoteDevice() {
+		return null;
+	}
+
+	public Node getEventNode() {
+		return null;

```

Comment:
DeviceFolder basically works as an editable folder. Its responsibility is maintaining edit-related actions. It also works as a point container. So it can't be abstract. DeviceNode is the only root folder which holds the device related info.